### PR TITLE
feat: add exposeHeader option

### PR DESCRIPTION
**Pull request by patrickmichalina**

### CODE REVIEW

**Comment by evanshortiss** [comment]
@patrickmichalina nice work. I can see why users could want this option 👍 

Would you be able to add a test in `tests/middleware.js`? Looking at the current test setup in `test/middleware.js` I see it's not ideal for making different instances of the middleware with different configurations so if it's a massive time sink, then no worries.

**Comment by patrickmichalina** [comment]
@evanshortiss I think is OK. The tests are a little weird but should be OK!

**Comment by patrickmichalina** [comment]
@evanshortiss So I actually allowed for customizing of the header since its basically the same change but more generalized.

File: README.md [hunk 1]
```diff
@@ -338,6 +338,8 @@ then this option is ignored.
 from the cache. By default _req.originalUrl_ is used as the key.
 * [Optional] statusCodeExpires - Useful if you want different status code responses to be cached for different
 durations.
+* [Optional] exposeHeader - Default responses use `x-expeditious-cache` as the header key. Set a new string value to
```
Comment (evanshortiss) [inline README.md]: Could I make one picky request? I feel like `exposeHeader` could be more descriptive if called `cacheStatusHeader`.

**Comment by evanshortiss** [comment]
@patrickmichalina awesome stuff, I have one comment but LGTM otherwise.

File: README.md (hunk 1 above)
Comment (patrickmichalina) [reply README.md]: Yeah, no problem

**Comment by patrickmichalina** [comment]
@evanshortiss OK! All done.

**Comment by evanshortiss** [comment]
@patrickmichalina released on npm as version 5.1.0

Thanks for the awesome PR 🎉

**Comment by patrickmichalina** [comment]
@evanshortiss awesome, thanks!

**Pull request merged**